### PR TITLE
rename the util class and added into new util package

### PR DESCRIPTION
**Pull request by nageshdivatagi**
created new package for util in identity service

### CODE REVIEW

File: src/main/java/com/identity/service/util/Util.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
-package com.identity.service;
+package com.identity.service.util;
 
-public class EsUtil {
+public class Util {
 
     public final static short ROLE_FIND_ALL_FLAG = 1;
     public final static short ROLE_FIND_ONE_FLAG = 1;
```
Comment (esoftdigital) [inline src/main/java/com/identity/service/util/Util.java:6]: using same flag for findbyId?

File: src/main/java/com/identity/service/service/RoleService.java [hunk 2]
```diff
@@ -35,7 +36,7 @@ public Roles findOne(short id) {
 	}
 
 	public Roles save(Roles role) {
-		role.setFlagId(EsUtil.ROLE_SAVE_FLAG);
+		role.setFlagId(Util.ROLE_SAVE_FLAG);
 
 		List<Roles> roleList = roleDaoService.procAddRole(role);
 		if (roleList.size() > 0)
```
Comment (esoftdigital) [inline src/main/java/com/identity/service/service/RoleService.java:42]: check for isEmpty

**Review by esoftdigital — CHANGES_REQUESTED** [review]

File: src/main/java/com/identity/service/service/RoleService.java (hunk 2 above)
Comment (nageshdivatagi) [reply src/main/java/com/identity/service/service/RoleService.java:42]: currently getById procedure is not available

**Review by nageshdivatagi — COMMENTED** [review]
currently getByID is not available in store Procedure

**Review by esoftdigital — APPROVED** [review]

**Pull request merged**